test/create-response-handler-test.ts

```diff
@@ -176,3 +176,121 @@ describe('The createResponseHandler function', () => {
     verifyNotCalled(onCancel)
   })
 })
+
```

Comment:
One thing that isn't currently tested is - do the the two different create[Identity]ResponseHandler() functions properly handle Scenario values that they shouldn't encounter?  I.e. does createResponseHandler() do something sensible if it encounters IDENTITY_VERIFIED, and does createIdentityResponseHandler() do something sensible if it encounters SUCCESS_MATCH, ACCOUNT_CREATION or NO_MATCH?

Not sure whether we should add tests for those; I think in other areas of passport-verify we have not really tried to handle 'impossible' situations in any specific way.